server/portal/apps/tickets/api/views.py

```diff
@@ -69,16 +70,26 @@ def post(self, request):
             metadata += "user_last_name:\n{}\n\n".format(data.get('last_name'))
 
         problem_description += "\n\n" + metadata
+        
+        if not request.user.is_authenticated:
+            recap_result = utils.get_recaptcha_verification(request)
+            if recap_result.get('success',False):
+                ticket_id = rt.create_ticket(subject=subject,
+                                         problem_description=problem_description,
+                                         requestor=email,
+                                         cc=cc,
+                                         attachments=attachments)
+                return JsonResponse({'ticket_id': ticket_id})
+            else:
+                raise ApiException('Invalid reCAPTCHA. Please try again.')
 
         ticket_id = rt.create_ticket(subject=subject,
-                                     problem_description=problem_description,
-                                     requestor=email,
-                                     cc=cc,
-                                     attachments=attachments)
-
+                                    problem_description=problem_description,
+                                    requestor=email,
+                                    cc=cc,
+                                    attachments=attachments)
```

Comment:
any reason for this line deletion?
```suggestion
                                    attachments=attachments)

```